app/webpacker/packs/application.js

```diff
@@ -32,6 +32,21 @@ document.addEventListener("turbolinks:load", () => {
     $("#prayer_categories").selectize({
         maxItems: 10
     });
+
+    $('#prayer_status').selectize({
+        create: true,
+        sortField: 'text'
+    });
+
+    $('#prayer_status').change(function() {
+        if ($("#curr_status").val() !== $(this).val()) {
+            $("#change_status_prompt").addClass("mb-3");
+            $("#change_status_prompt").html("<i>Why do you want to change status?</i>");
```

Comment:
@TimothyHou The status prompt appears on the top instead of the status part of the form. I'll changed it and added bootstrap style